flang/include/flang/Lower/PFTBuilder.h

```diff
@@ -64,7 +64,7 @@ class ReferenceVariantBase {
 
   template <typename B>
   constexpr BaseType<B> &get() const {
-    return std::get<Ref<B>> > (u).get();
+    return std::get<Ref<B>>> (u).get();
```

Comment:
clang-format is going a bit crazy here because this code I wrote a while ago is wrong (probably not instantiated for the const version): there should only be two `>` on the right.

Reply:
OK. Are you going to fix this or did you want me to delete the stray `>`?